res/css/style.css

```diff
@@ -574,3 +574,19 @@ body,
 .react-contextmenu-item.checked:hover {
   border-color: #fff;
 }
+
+.react-contextmenu-item--selected {
+  background-color: var(--blue-60);
```

Comment:
I find it weird that it's different from the one we get when hovering with the mouse (which is defined in https://github.com/firefox-devtools/profiler/blob/24f53d0408f7973cf2db9783a4ca62ec7da0873a/res/css/style.css#L506-L512). Maybe you can just add `.react-contextmenu-item--selected` to this ruleset?